### PR TITLE
Undo getSignatureStatus breaking change, add getSignatureStatuses

### PR DESCRIPTION
**Pull request by jstarry**
#### Problem
Made a breaking change to the return type and params for `getSignatureStatus` in a recent release and broke external clients

#### Summary of Changes
* Reintroduce the original behavior of `getSignatureStatus`
* Move recent API changes to `getSignatureStatuses`

Fixes #

### CODE REVIEW

**Comment by jstarry** [comment]
Here are the two commits which introduced the breaking changes for reference:
* https://github.com/solana-labs/solana/commit/1b8f9e75dd8981278f493d87af6e9345ba3a5ab7
* https://github.com/solana-labs/solana/commit/4b97e58cba2b15ea0ba6dc8996518b3bf0d9b7a4

**Comment by jstarry** [comment]
it's going to need a little hand holding to get through the backport merges.. `Custom` > `CustomError` for example

File: docs/src/apps/jsonrpc-api.md [hunk 1]
```diff
@@ -660,6 +661,38 @@ Returns the status of a given signature. This method is similar to [confirmTrans
...
 #### Parameters:
 
+* `<string>` - Signature of Transaction to confirm, as base-58 encoded string
+* `<object>` - (optional) [Commitment](jsonrpc-api.md#configuring-state-commitment)
+
+#### Results:
+
+An RpcResponse containing a JSON object consisting of an array of TransactionStatus objects.
+
+* `RpcResponse<object>` - RpcResponse JSON object with `value` field:
```
Comment (CriesofCarrots) [inline docs/src/apps/jsonrpc-api.md:671]: This needs to be removed

**Comment by jstarry** [comment]
I'm going to manually cherry pick instead of using mergify FYI

**Comment by solana-grimes** [comment]
:scream: New commits were pushed while the automerge label was present.

File: docs/src/apps/jsonrpc-api.md (hunk 1 above)
Comment (jstarry) [reply docs/src/apps/jsonrpc-api.md]: thx!

**Pull request merged**